speech/google/cloud/speech/client.py

```diff
@@ -199,6 +223,98 @@ def sync_recognize(self, sample, language_code=None,
         else:
             raise ValueError('result in api should have length 1')
 
+    def stream_recognize(self, sample, language_code=None,
+                         max_alternatives=None, profanity_filter=None,
+                         speech_context=None, single_utterance=False,
+                         interim_results=False):
+        """Streaming speech recognition.
+
+        .. note::
+            Streaming recognition requests are limited to 1 minute of audio.
+
+            See: https://cloud.google.com/speech/limits#content
+
+        :type sample: :class:`~google.cloud.speech.sample.Sample`
+        :param sample: Instance of ``Sample`` containing audio information.
+
+        :type language_code: str
+        :param language_code: (Optional) The language of the supplied audio as
+                              BCP-47 language tag. Example: ``'en-GB'``.
+                              If omitted, defaults to ``'en-US'``.
+
+        :type max_alternatives: int
+        :param max_alternatives: (Optional) Maximum number of recognition
+                                 hypotheses to be returned. The server may
+                                 return fewer than maxAlternatives.
+                                 Valid values are 0-30. A value of 0 or 1
+                                 will return a maximum of 1. Defaults to 1
+
+        :type profanity_filter: bool
+        :param profanity_filter: If True, the server will attempt to filter
+                                 out profanities, replacing all but the
+                                 initial character in each filtered word with
+                                 asterisks, e.g. ``'f***'``. If False or
+                                 omitted, profanities won't be filtered out.
+
+        :type speech_context: list
+        :param speech_context: A list of strings (max 50) containing words and
+                               phrases "hints" so that the speech recognition
+                               is more likely to recognize them. This can be
+                               used to improve the accuracy for specific words
+                               and phrases. This can also be used to add new
+                               words to the vocabulary of the recognizer.
+
+        :type single_utterance: boolean
+        :param single_utterance: [Optional] If false or omitted, the recognizer
```

Comment:
The other places this occurs (via copy-paste) also should be updated to whatever you decide on